### ISSUE TITLE
[SPK][Spike] Tracking external applications

### ISSUE THREAD

**Issue by mtarng** [microsoft/bedrock#824]
**As a:**
Developer

**I want:**
to be able to use external applications and have my deployments managed by the pipelines.

**Additional context:**
If I have a bedrock project that is dependent on a service application, how would this be done? ie. I need to deploy a Mongo or a Postgres container on my cluster along side my other microservices.

Additionally, how would service introspection be able to track the deployment process?

**Does this require updates to documentation?:**
yes

**Comment by andrebriggs** [comment]
@mtarng I'm understanding your scenario as "What if you have additional applications in the HLD that are essentially 3rd party?" How do you track them  in the service introspection tool?

Correct interpretation?

**Comment by mtarng** [comment]
@andrebriggs yes, that's correct. Though these external applications may be defined inside one or more application monorepositories. So this also covers how would we may manage and track their deployments, and how would we manage conflicts between dependencies from other application repositories.

**Comment by andrebriggs** [comment]
> @andrebriggs yes, that's correct. Though these external applications may be defined inside one or more application monorepositories. So this also covers how would we may manage and track their deployments, and how would we manage conflicts between dependencies from other application repositories.
> 
> @bnookala for context

@bnookala context?

**Comment by andrebriggs** [comment]
As discussed @bnookala @mtarng will re-work this issue
